src/switchroot/ostree-mount-util.h

```diff
@@ -114,6 +120,46 @@ read_proc_cmdline_key (const char *key)
   return ret;
 }
 
+static inline char *
+get_aboot_root_slot (void)
+{
+  char __attribute__ ((cleanup (free_char))) *slot_suffix
+      = read_proc_cmdline_key ("androidboot.slot_suffix");
+  if (slot_suffix[1] == 'a')
+    return strdup ("/ostree/root.a");
+  else if (slot_suffix[1] == 'b')
+    return strdup ("/ostree/root.b");
+
+  errx (EXIT_FAILURE, "androidboot.slot_suffix invalid: %s", slot_suffix);
+
+  return NULL;
+}
+
+static inline char *
+move (char **to_move)
```

Comment:
This is typically called steal_pointer()

Reply:
move is a C++ism for the same thing, can change to steal_pointer

Reply:
Well, it is g_steal_pointer() in glib which is used in the rest of ostree.